File: Level 6/HW6.py
x=1                #1
while x<=10:
    print(x)
    x=x+1

y=10                #2
while y>=1:
    print(y)
    y=y-1

#while loop-პროცესი რომელიც მიმდინარეობს მანამ სანამ მოცემული პირობა შეიცვლება, თუ არა და ის სამუდამოდ გაგრძელდება   #3

z = 'python123'                                 #4 აქ, როდესაც თავიდან მომხმარებელი წერს პაროლს, და ის არ შეესაბამება,   
a = input('Input a password: ')                 # იმის შემდეგ რაც Password is incorrect დაიპრინტება,
                                                # პროგრამა მას თავიდან სტხოვს მას თვით while loopში და თავიდან აწერინებს.  
while a != z:                                   # ეს გაგრძელდება სანამ ის არ შეიყვანს სწორ პაროლს
    print('Password is incorrect')
    a = input('Try again: ')

print('Password is correct')

b = int(input('Enter a number: '))  #5 
c = 1
sum = 0
while c <= b:
    sum += c
    c += 1
print(sum)     
    

File: Level 63/Homework_63/HW6.js
// შექმენი ობიექტი calculator და დაამატე თვისება n. შექმენი მეთოდი, რომელიც for loop-ით ითვლის 1–დან n-მდე ჯამს.
let calculator = {
    n: 5,

    sum: function() {
        let total = 0;

        for (let i = 1; i <= this.n; i++) {
            total += i;
        }

        return total;
    }
};

console.log(calculator.sum());